quinn-proto/src/connection/streams.rs

```diff
@@ -435,6 +436,7 @@ impl Streams {
         buf: &mut Vec<u8>,
         pending: &mut Retransmits,
         sent: &mut Retransmits,
+        stats: &mut ConnectionStats,
```

Comment:
Should this just be `FrameTransmissionStats`?

Reply:
Could be. Although might be more in the future. I have no strong opinion. Since it’s internal it could always be changed

Reply:
Given that this function specifically deals with frames, I'd err on the side of the more concise and narrowly-scoped approach, though it's not too important.